selfdrive/car/volkswagen/carstate.py

```diff
@@ -41,9 +42,10 @@ def update(self, pt_cp, cam_cp, ext_cp, trans_type):
     ret.yawRate = pt_cp.vl["ESP_02"]["ESP_Gierrate"] * (1, -1)[int(pt_cp.vl["ESP_02"]["ESP_VZ_Gierrate"])] * CV.DEG_TO_RAD
 
     # Verify EPS readiness to accept steering commands
+    # To handle cars without factory Lane Assist, treat FAULT like INITIALIZING for three seconds after startup
     hca_status = self.hca_status_values.get(pt_cp.vl["LH_EPS_03"]["EPS_HCA_Status"])
-    ret.steerFaultPermanent = hca_status in ("DISABLED", "FAULT")
-    ret.steerFaultTemporary = hca_status in ("INITIALIZING", "REJECTED")
+    ret.steerFaultPermanent = hca_status == "DISABLED" or (hca_status == "FAULT" and self.frame >= 300)
```

Comment:
this is less than the [controls initializing time](https://github.com/commaai/openpilot/blob/22408781f7d7df5ff0395f80c1b15d6407acd0ff/selfdrive/controls/controlsd.py#L416), perhaps we can consider faults in there too

Reply:
or you can mark canValid as false

Reply:
I slid the timer out to compensate for worst case controls init and commented why. There's no real harm in doing so. I prefer that over the later troubleshooting headache I'd invite by tampering with canValid outside normal paths.

Reply:
I looked more closely at your proposal, and as far as I can tell, we don't [signal boardd to set our safety mode](https://github.com/commaai/openpilot/blob/20c140b10eef52b6f6d6b9e142ed944264865bac/selfdrive/controls/controlsd.py#L425) or begin [sending CAN messages from carcontroller](https://github.com/commaai/openpilot/blob/20c140b10eef52b6f6d6b9e142ed944264865bac/selfdrive/controls/controlsd.py#L720-L723) until after CS initialization. For purposes of this PR, the EPS won't start recovering until it starts seeing openpilot CAN traffic, so holding down controls init would be counterproductive.

Reply:
Do you care to see the 600 restated as (6. / DT_CTRL)? Seems weird to care about DT_CTRL inside a car port, but you're doing it in comma body...